acceptable/_service.py

```diff
@@ -159,12 +160,19 @@ def api(self,
             introduced_at,
             options,
             undocumented=undocumented,
+            deprecated_at=deprecated_at,
             location=location,
         )
         self.metadata.register_api(self.name, self.group, api)
         return api
 
-    def django_api(self, name, introduced_at, undocumented=False, **options):
+    def django_api(
+            self,
+            name,
+            introduced_at,
+            undocumented=False,
+            deprecated_at=None,
+            **options):
         """Add an django API handler to the service.
```

Comment:
I know you didn't touch this but, since you're here can you s/an django/a Django/